misk/web/tabs/config/src/components/ConfigComponent.tsx

```diff
@@ -0,0 +1,86 @@
+// tslint:disable-next-line:no-var-requires
+const YAML = require("json2yaml")
+import * as React from "react"
+import styled from "styled-components"
+import { IConfigFile } from "../containers/TabContainer"
+
+interface IConfigProps {
+  files: IConfigFile[]
+  status: string
+}
+
+const Container = styled.div`
+  margin-left: 180px;
+  margin-top: 20px;
+`
+
+export default class ConfigComponent extends React.PureComponent<IConfigProps> {
+  indent(spaces: number) {
+    let final = ""
```

Comment:
is `final` a convention? We use `result` in Kotlin  . . . (`final` is surprising ’cause it gets replaced)

Reply:
This entire part of the ConfigComponent is still very rough/in progress. Trying to figure out how to parse the Jackson JsonNode.toString back to YAML using NPM packages or basic parsing.